opennms-container/minion/confd/templates/confd.system.properties.tmpl

```diff
@@ -0,0 +1,10 @@
+{{$snmp2In1Key := "/snmp/snmp4j/org.opennms.snmp.snmp4j.allowSNMPv2InV1" -}}
```

Comment:
Should be be strongly modeling all of the system properties, or try and keep this generic from a template perspective?

Reply:
Generic will let us be a lot more flexible, will update.